src/main/java/com/ibm/devops/connect/CRPipeline/UploadBuild.java

```diff
@@ -262,5 +331,25 @@ public String getDisplayName() {
         public boolean isApplicable(Class<? extends AbstractProject> jobType) {
             return true;
         }
+
+        public ListBoxModel doFillInstanceBaseUrlItems(@QueryParameter String currentInstanceBaseUrl) {
```

Comment:
This looks similar to what was in another file as well.  Is it possible it can be done in a shared place?